Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -44,6 +49,49 @@ static gboolean webKitMediaOpenCDMDecryptorHandleKeyResponse(WebKitMediaCommonEn
 static gboolean webKitMediaOpenCDMDecryptorDecrypt(WebKitMediaCommonEncryptionDecrypt*, GstBuffer*, GstBuffer*, unsigned, GstBuffer*);
 static void webKitMediaOpenCDMDecryptorReceivedProtectionEvent(WebKitMediaCommonEncryptionDecrypt*, unsigned);
 
+static GstStaticPadTemplate sinkTemplate = GST_STATIC_PAD_TEMPLATE("sink",
+    GST_PAD_SINK,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS(
+    "application/x-cenc, original-media-type=(string)video/webm, protection-system=(string)" WEBCORE_CDMFACTORY_SYSTEM_UUID "; "
```

Comment:
You don't need MSE and non-MSE UUIDs. The same should be used. These are the capabilities accepted by the decryptor and it needs to be able to accept just the three UUIDs once. The decryptor creation, both in GStreamer and in WebKit, should already be able to detect this.

Reply:
But the problem here is that I can only return one in the protection Id in the message. OpenCDM allows for, potentially, all CDMs. but than can not be returned from in the message. I suggest to do this in a next round.
So do agree that that should work, but this requires more modifications ( and I prefer to take it one step at a time.
https://github.com/WebPlatformForEmbedded/WPEWebKit/blob/pwielders/gstreamer-decryption-refactor/Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp#L123
Here I can only fill in 1 protectionSystemId, in OpenCDM context this could be multiple and personally I don like to "abuse" one of the existing (PlayReady/WideVine/Clearkey) properties to use in this spot.